lib/rucio/core/replica.py

```diff
@@ -3195,6 +3195,30 @@ def get_replica_atime(
     return session.execute(stmt).scalar_one()
 
 
+@read_session
+def get_replica_updated_at(replica: dict[str, Any], *, session: "Session") -> Optional[datetime]:
```

Comment:
`scalar_one()` raises an exception if the query does not match exactly one row. As such, the use of `Optional` is wrong here.